<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2KLPDBI" failures="1" tests="3" name="Submit Order Test" time="42.4" errors="0" timestamp="2024-08-09T20:01:17 IST">
  <testcase classname="TestCases.tests.SubmitOrderTest" name="submitOrder" time="13.72">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.tests.SubmitOrderTest.submitOrder(SubmitOrderTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase classname="TestCases.tests.SubmitOrderTest" name="submitOrder" time="3.797"/>
  <testcase classname="TestCases.tests.SubmitOrderTest" name="OrderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test -->
